### ISSUE TITLE
PHPUnit_Framework_Exception: Codeception\Lib\InnerBrowser::getFormFromCrawler(): ID identity already defined

### ISSUE THREAD

**Issue by chrissound** [Codeception/Codeception#1986]
Hi there, I'm running into a test failure as well as the exception above being shown. 

```
<?php
$I = new AcceptanceTester($scenario);
$I->wantTo('login');
$I->amOnPage('/');
$I->see('User login');
$I->see('Email');
$I->see('Password');
$I->click('submit');
```
```
Codeception PHP Testing Framework v2.0.13
Powered by PHPUnit 4.6.4 by Sebastian Bergmann and contributors.

Unit Tests (0) ------------------------------
---------------------------------------------

Functional Tests (0) ------------------------
---------------------------------------------

Acceptance Tests (1) --------------------------------------------------------------------------------------
Login (WelcomeCept)                                                                                  Error
-----------------------------------------------------------------------------------------------------------


Time: 236 ms, Memory: 11.50Mb

There was 1 error:

---------
1) Failed to login in WelcomeCept (/home/chris/NetBeansProjects/x/tests/acceptance/WelcomeCept.php)
Couldn't click "submit":
PHPUnit_Framework_Exception: Codeception\Lib\InnerBrowser::getFormFromCrawler(): ID identity already defined

Scenario Steps:
5. I click "submit"
4. I see "Password"
3. I see "Email"
2. I see "User login"
1. I am on page "/"

#1  Codeception\Lib\InnerBrowser->click
#2  /home/chris/NetBeansProjects/x/tests/acceptance/AcceptanceTester.php:222
#3  /home/chris/NetBeansProjects/x/tests/acceptance/WelcomeCept.php:8
#4  Codeception\TestCase\Cept->testCodecept
#5  /home/chris/Utils/codeception/codecept.phar:7
                                   
FAILURES!                          
Tests: 1, Assertions: 3, Errors: 1.
```

**Comment by DavertMik** [comment]
I just released 2.0.14 which should fix this issue. Please update

**Comment by jcarter-akt** [comment]
I'm on the latest version of Codeception and am receiving this error when trying to access a submit button:

[PHPUnit_Framework_Exception] DOMDocument::importNode(): ID scroll-to-1 already defined

It seemed related so felt it was worth posting here.
